build.xml

```diff
@@ -771,6 +771,9 @@
         <dependency groupId="org.junit" artifactId="junit-bom" version="5.6.0" type="pom"/>
         <dependency groupId="org.assertj" artifactId="assertj-core"/>
         <dependency groupId="org.awaitility" artifactId="awaitility"/>
+        <dependency groupId="org.apache.lucene" artifactId="lucene-core" version="7.5.0" />
```

Comment:
@justinchuch Can you review this commit please. It contains the changes needed in the Cassandra test infrastructure for SAI testing.